elasticsearch-monitoring/README.md

```diff
@@ -0,0 +1,68 @@
+# es-monitoring
+
+A Helm chart for monitoring Elasticsearch via Prometheus and Grafana. This chart will deploys the [elasticsearch-exporter](https://github.com/justwatchcom/elasticsearch_exporter) and configures Prometheus (via the [Operator](https://github.com/coreos/prometheus-operator)) for scraping the exposed metrics, sets up some alerts and Grafana dashboards (todo).
+
+## TL;DR
+
+```shell
+helm install skyscrapers/elasticsearch-monitoring -f values.yaml
+```
+
+## Prerequisites
+
+- Kubernetes 1.6+
```

Comment:
I think prometheus operator will also be a prerequisite ;)

Reply:
Fair point, done.